conda/gateways/disk/create.py

```diff
@@ -178,6 +178,30 @@ def create_hard_link_or_copy(src, dst):
         shutil.copy2(src, dst)
 
 
+def _do_softlink(src, dst):
+    if islink(src):
+        symlink(src, dst)
```

Comment:
@mingwandroid I added the two above lines.  Are you fine with this still?  If not, we might have to tighten down `is_executable()` a lot more.